src/compute/src/sink/materialized_view_v2.rs

```diff
@@ -855,6 +883,25 @@ mod write {
                     self.batch_description = None;
                 }
             }
+
+            self.maybe_force_consolidation();
+        }
+
+        /// If the current consolidation request has become applicable, apply it.
+        fn maybe_force_consolidation(&mut self) {
+            let Some(request) = &self.force_consolidation_after else {
+                return;
+            };
+
+            let desired_frontier = self.desired_frontiers.frontier();
+            let persist_frontier = self.persist_frontiers.frontier();
+            if PartialOrder::less_than(request, desired_frontier)
+                && PartialOrder::less_than(request, persist_frontier)
```

Comment:
I'm not sure I fully understand this: Doesn't this apply consolidation for every `maybe_*` call once the frontiers are past `self.force_consolidation_after`? What I don't yet understand is whether this is the desired behavior or a potential regression.

Reply:
No, you found a bug! This is supposed to remove the request (by replacing it with `None`) once it has been fulfilled, but it doesn't do that.

Reply:
Fixed.